custom_components/hypervolt_charger/manifest.json

```
@@ -7,6 +7,6 @@
   "integration_type": "device",
   "iot_class": "cloud_polling",
   "issue_tracker": "https://github.com/gndean/home-assistant-hypervolt-charger/issues",
-  "requirements": ["websockets==11.0.3", "aiofiles==24.1.0"],
+  "requirements": ["websockets>=11.0.3", "websockets<=14.0", "aiofiles==24.1.0"],
```

```suggestion
  "requirements": ["websockets>=11.0.3,<14", "aiofiles==24.1.0"],
```
You can commit this suggestion directly from github to your branch